app/src/main/res/values/strings.xml

```diff
@@ -32,7 +32,7 @@
   <string name="patientsearchresults_register_patient_rationale_for_empty_state">Patient is not registered</string>
   <string name="patientsearchresults_register_patient">Register as a new patient</string>
   <string name="patientsearchresults_register_patient_for_empty_state">Register patient</string>
-  <string name="patientsearchresults_item_last_bp">Last BP:</string>
+  <string name="patientsearchresults_item_last_bp">Last Seen:\u0020</string>
```

Comment:
Please create a new string for `recent_patient` and revert the string for `patientsearchresults`.

Reply:
This has created a bug where `LAST BP` is now called `LAST SEEN` in Patient search results.

Reply:
resolved